src/utils/awsAccess.ts

```diff
@@ -0,0 +1,96 @@
+import jwt from "@tsndr/cloudflare-worker-jwt";
+import { v4 as uuidv4 } from "uuid";
+import { env } from "../typeDefinitions/default.types";
+import config from "../../config/config";
+import { discordTextResponse } from "./discordResponse";
+import { DISCORD_BASE_URL, AWS_IAM_SIGNIN_URL } from "../constants/urls";
+
+export async function processAWSAccessRequest(
+  discordUserId: string,
+  awsGroupId: string,
+  env: env,
+  TraceId: string,
+  channelId: number
+) {
+  const authToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 2 },
+    env.BOT_PRIVATE_KEY,
+    { algorithm: "RS256" }
+  );
+
+  try {
+    const base_url = config(env).RDS_BASE_API_URL;
+    const requestData = {
+      groupId: awsGroupId,
+      userId: discordUserId,
+    };
+
+    const url = `${base_url}/aws-access/`;
```

Comment:
Can you create the URL in constant it self?

Reply:
And please share the doc for using it herE?

Reply:
do we mean it like 
`const url = ${base_url}/aws-access` this way we have it stored?

Reply:
Didn't get this which doc?
> And please share the doc for using it herE?

Reply:
Please share the AWS API docs like, of what they are doing?

Reply:
This is our website backend API, will add the comments there.